platform/email/platform-email-impl/src/main/java/org/codice/ddf/platform/email/impl/SmtpClientImpl.java

```diff
@@ -168,23 +168,27 @@ public Session createSession() {
   @Override
   public Future<Void> send(Message message) {
     notNull(message, "message must be non-null");
+    try {
+      securityLogger.audit(
+          "Sending email: recipient={} subject={}",
+          Arrays.toString(message.getAllRecipients()),
+          message.getSubject());
+    } catch (MessagingException e) {
+      // ignore
```

Comment:
✏️ If we cannot create an audit log message, then I would suggest we do what the other catch block below is doing.